.editorconfig

```diff
@@ -1,5 +1,16 @@
 root = true
 
-[*.js]
-indent_style=tab
-trim_trailing_whitespace=true
\ No newline at end of file
+[*]
+indent_style = tab
+indent_size = 2
+end_of_line = lf
+charset = utf-8
```

Comment:
A good idea could add an exception only for the files in this folder. [utf-8 with BOM is not recommended and encouraged](http://stackoverflow.com/questions/2223882/whats-different-between-utf-8-and-utf-8-without-bom#answer-2223926). 

Just to check: Are we sure about it?